pyomo/contrib/mindtpy/util.py

```diff
@@ -109,9 +109,9 @@ def var_bound_add(solve_data, config):
                     continue
                 elif not var.has_lb():
                     if var.is_integer():
-                        var.setlb(-config.integer_var_bound)
+                        var.setlb(-config.integer_var_bound - 1)
                     else:
-                        var.setlb(-config.continuous_var_bound)
+                        var.setlb(-config.continuous_var_bound - 1)
```

Comment:
Technically, you should not subtract 1 for the continuous bound.  The "`- 1`' only accounts for 2's complement in integers and does not apply (in the same way) to floating point numbers.